docs/HOW_TO_TEST.md

```diff
@@ -88,6 +88,45 @@ Simply run:
 molecule destroy -s <scenario name>
 ```
 
+### Running with some enviroment variables set
```

Comment:
This is not needed. We can simply set env variables using export on the system and use Kraft/arm64 mode. Please remove this section from the docs. We use this passing of file system for automating the tests and decreasing manual work.

Reply:
okay will remove this one